libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -172,6 +172,14 @@ class BreezSDKMapper {
             "bolt11": invoicePaidDetails.bolt11
         ]
     }
+
+    static func dictionaryOf(paymentFailedData: PaymentFailedData) -> [String: Any?] {
+        return [
+            "error": paymentFailedData.error,
+            "bolt11": dictionaryOf(lnInvoice: paymentFailedData.bolt11),
```

Comment:
If bolt11 is optional then `dictionaryOf(lnInvoice)` needs to handle the nil case.

Reply:
Right, missed that. Fixed.